templates/ims/ims.yaml

```diff
@@ -41,20 +41,23 @@ spec:
         - name: {{ .Chart.Name }}-ims
           image: "{{ .Values.ims.image.repository }}:{{ .Values.ims.image.tag }}"
           imagePullPolicy: {{ .Values.ims.image.pullPolicy }}
-          livenessProbe:
-            initialDelaySeconds: 600
-            failureThreshold: 30
-            periodSeconds: 8
-            httpGet:
-              path: /
-              port: 8080
-          readinessProbe:
-            failureThreshold: 4
-            initialDelaySeconds: 150
-            periodSeconds: 10
-            httpGet:
-              path: /
-              port: 8080
+          securityContext:
+            allowPrivilegeEscalation: {{ .Values.containerSecurity.allowPrivilegeEscalation }}
+            privileged: {{ .Values.containerSecurity.privileged }}
+          # livenessProbe:
+          #   initialDelaySeconds: 600
+          #   failureThreshold: 30
+          #   periodSeconds: 8
+          #   httpGet:
+          #     path: /
+          #     port: 8080
+          # readinessProbe:
+          #   failureThreshold: 4
+          #   initialDelaySeconds: 150
+          #   periodSeconds: 10
+          #   httpGet:
+          #     path: /
+          #     port: 8080
```

Comment:
We have revision control, no need to save commented out things.

Also, we want the `readinessProbe` at least.

Reply:
Yes, we might need `initialDelaySeconds` of `300` or `600`.